smoke/README.md

```diff
@@ -6,3 +6,20 @@ This is a collection of generic smoke tests to validate the basic functionality
 
 For details check the main [README.md](../README.md#run-the-test-against-your-aem-cloud-service-author-and-publish-tiers)
 of this repository.
+
+## Exceptions
+
+### Test - PublishEndToEndIT
+
+| Exceptions           | Error Code                  | Reason                                                             |
+|----------------------|-----------------------------|--------------------------------------------------------------------|
+| PublishException     | PAGE_NOT_AVAILABLE          | Activated page not available on publish                            |
+|                      | PAGE_AVAILABLE              | Deactivated page still available on publish                        |
+| ReplicationException | QUEUE_BLOCKED               | Replication queue blocked before test                              |
+|                      | ACTIVATION_REQUEST_FAILED   | Replication action failed                                          |
+|                      | DEACTIVATION_REQUEST_FAILED | Deactivation action failed                                         |
+|                      | ITEM_NOT_REPLICATED         | Item to be replicated still available on agent queues              |
+|                      | REPLICATION_UNAVAILABLE     | Replication agents not available before test                       |
```

Comment:
Suggest to `s/REPLICATION_UNAVAILABLE/REPLICATION_NOT_AVAILABLE` to align with the other error code.